rootfs-builder/template/config_template.sh

```diff
@@ -6,3 +6,6 @@
 OS_VERSION=${OS_VERSION:-DEFAULT_VERSION}
 
 PACKAGES="systemd iptables udevlib.so"
+
+INIT_PROCESS=systemd
```

Comment:
Might be worth adding a comment saying we expect this to be either "kata-agent" or "systemd".